docs/src/main/tut/docs/stack/cache.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: docs
```

Comment:
Comments on the review of the previous file regarding the example, explanations etc.. should also apply on this one.